ui/src/components/Header.js

```diff
@@ -58,6 +58,8 @@ const ConsoleMeHeader = () => {
           title={user.user}
         />
       );
+    } else if (user?.user) {
```

Comment:
What about always showing username, and also showing avatar if we have it?

For bonus points, we could fall back to Gravatar for avatars with `https://www.gravatar.com/avatar/<md5-of-email>`.  I'm https://www.gravatar.com/avatar/3df5aeaf94497a26b28fd761b2261177

Reply:
Good call, done and done with fallback to Gravatar